airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -139,7 +140,6 @@ public final ReplicationOutput run(final StandardSyncInput syncInput, final Path
     destinationConfig.setCatalog(mapper.mapCatalog(destinationConfig.getCatalog()));
 
     final ThreadedTimeTracker timeTracker = new ThreadedTimeTracker();
-    final long startTime = System.currentTimeMillis();
```

Comment:
This is the only deletion in the entire file. We can replace this with `timeTracker.trackReplicationStartTime()` so this is no longer needed.